meson.build

```diff
@@ -161,7 +161,7 @@ else
   r2_libdir = join_paths(r2_prefix, get_option('libdir'))
   r2_incdir = join_paths(r2_prefix, get_option('includedir'), 'libr')
   r2_datdir = join_paths(r2_prefix, get_option('datadir'))
-  r2_wwwroot = join_paths(r2_datdir, 'radare2', r2_version, 'www')
+  r2_wwwroot = join_paths(get_option('datadir'), 'radare2', r2_version, 'www')
```

Comment:
missing space before (

Reply:
It is `meson.build`. We use own codestyle.